src/actions/sources/select.js

```diff
@@ -227,3 +247,17 @@ export function jumpToMappedSelectedLocation() {
     await dispatch(jumpToMappedLocation(location));
   };
 }
+
+/**
+ * Updates the file search results if a file search is open
+ */
+function clearFileSearch() {
+  return async ({ dispatch, getState }: ThunkArgs) => {
+    const isFileSearchOpen = getActiveSearch(getState()) === "file";
+    const fileSearchQuery = getFileSearchQuery(getState());
+    if (isFileSearchOpen && fileSearchQuery) {
+      const editor = getEditor();
+      dispatch(searchContents(fileSearchQuery, editor));
+    }
+  };
+}
```

Comment:
lets move this to a different action file that is focused on search

Reply:
I moved it to the ui.js actions file because that has the similar actions `closeActiveSearch` and `setActiveSearch`.  But it if you prefer it being in the file-seach.js actions file I could move it there instead.  I also renamed the action to `updateActiveFileSeach` since I think that better describes what it does rather than `clearFileSearch`